docs/sources/tempo/release-notes/v2-2.md

```diff
@@ -0,0 +1,151 @@
+
+To learn more about this API, refer to the[ metrics summary API documentation.](/docs/grafana-cloud/data-configuration/traces/metrics-summary-api/)
+
+This work is represented in multiple PRs: [2368](https://github.com/grafana/tempo/pull/2368), [2418](https://github.com/grafana/tempo/pull/2418), [2424](https://github.com/grafana/tempo/pull/2424), [2442](https://github.com/grafana/tempo/pull/2442), [2480](https://github.com/grafana/tempo/pull/2480), [2481](https://github.com/grafana/tempo/pull/2481), [2501](https://github.com/grafana/tempo/pull/2501), [2579](https://github.com/grafana/tempo/pull/2579), and [2582](https://github.com/grafana/tempo/pull/2582).
+
+
+### Other enhancements
+
+* Tempo’s [tag values]({{< relref "../api_docs#search-tag-values" >}}) and [tag names]({{< relref "../api_docs#search-tags" >}}) APIs now support filtering [PR [2253](https://github.com/grafana/tempo/pull/2253)]. This lets you retrieve all valid attribute values and names given certain criteria. For example, you can get a list of values for the attribute `namespace` seen on spans with attribute `resource=A.` This feature is off by default; to enable, configure `autocomplete_filtering_enabled`. ([documentation]({{< relref "../api_docs" >}})). Grafana’s autocomplete can make use of this filtering capability to provide better suggestions starting in v10.2 [[PR 67845]](https://github.com/grafana/grafana/pull/67845).
+
+* Tempo’s metrics-generator now supports span filtering. Setting up filters allows you to compute metrics over the specific spans you care about, excluding others. It also can reduce the cardinality of generated metrics, and therefore the cost of storing those metrics in a Prometheus-compatible TSDB. ([documentation]({{< relref "../metrics-generator/span_metrics#filtering" >}})) [PR [2274](https://github.com/grafana/tempo/pull/2274)]
+
+* Tempo’s metrics-generator can now detect virtual nodes ([documentation]({{< relref "../metrics-generator/service_graphs#virtual-nodes" >}})) [PR [2365](https://github.com/grafana/tempo/pull/2365)]. As a result, you’ll now see these virtual nodes represented in your service graph. For more information, refer to the [virtual nodes documentation]({{< relref "../metrics-generator/service_graphs#virtual-nodes" >}}).
+
+
+## Upgrade considerations
+
+When [upgrading](http://localhost:3002/docs/tempo/latest/setup/upgrade/) to Tempo 2.2, be aware of these breaking changes:
+
+* Convert metrics generator from deployment to a statefulset in JSonnet. Refer to the PR for seamless migration instructions. [PRs [#2533](https://github.com/grafana/tempo/pull/2533), [#2467](https://github.com/grafana/tempo/pull/2647)]
```

Comment:
can we just add a quick note "Impacts jsonnet users only" or something?

Reply:
```suggestion
* JSonnet users only: Convert metrics generator from deployment to a statefulset in JSonnet. Refer to the PR for seamless migration instructions. [PRs [#2533](https://github.com/grafana/tempo/pull/2533), [#2467](https://github.com/grafana/tempo/pull/2647)]
```